sgv2-restapi/src/main/java/io/stargate/sgv2/restsvc/resources/schemas/Sgv2ColumnsResourceImpl.java

```diff
@@ -0,0 +1,282 @@
+        keyspaceName,
+        tableName,
+        (tableDef) -> {
+          Column.Kind kind =
+              columnDefinition.getIsStatic() ? Column.Kind.STATIC : Column.Kind.REGULAR;
+          Column columnDef =
+              ImmutableColumn.builder()
+                  .name(columnName)
+                  .kind(kind)
+                  .type(columnDefinition.getTypeDefinition())
+                  .build();
+          String cql =
+              new QueryBuilder()
+                  .alter()
+                  .table(keyspaceName, tableName)
+                  .addColumn(columnDef)
+                  .build();
+          blockingStub.executeQuery(
+              QueryOuterClass.Query.newBuilder()
+                  .setParameters(parametersForLocalQuorum())
+                  .setCql(cql)
+                  .build());
+
+          return jaxrsResponse(Response.Status.CREATED)
+              .entity(Collections.singletonMap("name", columnName))
+              .build();
+        });
+  }
+
+  @Override
+  public Response getOneColumn(
+      StargateGrpc.StargateBlockingStub blockingStub,
+      String keyspaceName,
+      String tableName,
+      String columnName,
+      boolean raw,
+      HttpServletRequest request) {
+    if (isStringEmpty(keyspaceName)) {
+      return jaxrsBadRequestError("keyspaceName must be provided").build();
+    }
+    if (isStringEmpty(tableName)) {
+      return jaxrsBadRequestError("table name must be provided").build();
+    }
+    if (isStringEmpty(columnName)) {
+      return jaxrsBadRequestError("columnName must be provided").build();
+    }
+    return callWithTable(
+        blockingStub,
+        keyspaceName,
+        tableName,
+        (tableDef) -> {
+          Sgv2ColumnDefinition column = findColumn(tableDef, columnName);
+          if (column == null) {
+            return jaxrsBadRequestError(
+                    String.format("column '%s' not found in table '%s'", columnName, tableName))
+                .build();
+          }
+          final Object payload = raw ? column : new Sgv2RESTResponse(column);
+          return jaxrsResponse(Response.Status.OK).entity(payload).build();
+        });
+  }
+
+  @Override
+  public Response updateColumn(
+      StargateGrpc.StargateBlockingStub blockingStub,
+      String keyspaceName,
+      String tableName,
+      String columnName,
+      Sgv2ColumnDefinition columnUpdate,
+      HttpServletRequest request) {
+    if (isStringEmpty(keyspaceName)) {
+      return jaxrsBadRequestError("keyspaceName must be provided").build();
+    }
+    if (isStringEmpty(tableName)) {
+      return jaxrsBadRequestError("table name must be provided").build();
+    }
+    if (isStringEmpty(columnName)) {
+      return jaxrsBadRequestError("columnName must be provided").build();
+    }
+    return callWithTable(
+        blockingStub,
+        keyspaceName,
+        tableName,
+        (tableDef) -> {
+          final String newName = columnUpdate.getName();
```

Comment:
short-circuit if new name is the same as old?

Reply:
Will do, sure.